### PR TITLE
Fix mkdocs-techdocs-core version

### PR DESCRIPTION
**Pull request by dweber019**
## Hey, I just made a Pull Request!
Probably a old version as in your docker image you use https://github.com/backstage/techdocs-container/blob/main/Dockerfile#L23

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Review by iamEAP — COMMENTED** [review]
Thanks @dweber019!  Looks like the only failing check is the `DCO`.  Could you [check out the instructions here](https://github.com/backstage/backstage/pull/8625/checks?check_run_id=4622339422) and re-push?

File: docs/features/techdocs/getting-started.md [hunk 1]
```diff
@@ -230,7 +230,7 @@ You can do so by including the following lines in the last step of your
 
 ```Dockerfile
 RUN apt-get update && apt-get install -y python3 python3-pip
-RUN pip3 install mkdocs-techdocs-core==0.0.16
+RUN pip3 install mkdocs-techdocs-core==0.2.1
```
Comment (iamEAP) [inline docs/features/techdocs/getting-started.md:233]: I believe we also just released `0.2.2` a bit ago.  While you're updating things for DCO, could you also update the pinned version?

Comment (dweber019) [reply docs/features/techdocs/getting-started.md]: done

**Review by iamEAP — APPROVED** [review]
Thanks, @dweber019!

**Pull request merged**